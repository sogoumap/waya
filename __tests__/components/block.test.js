import 'babel-polyfill';
import { mount } from '@vue/test-utils';
import Block from '../../src/components/block';

describe('SnapShot Cases',()=>{
  test('Block',()=>{
    const Wrapper = mount(Block);
    expect(Wrapper).toMatchSnapshot();
  });
  test('title',()=>{
    const Wrapper = mount(Block.Title);
    expect(Wrapper).toMatchSnapshot();
  });
  test('header',()=>{
    const Wrapper = mount(Block.Header);
    expect(Wrapper).toMatchSnapshot();
  });
  test('content',()=>{
    const Wrapper = mount(Block.Content);
    expect(Wrapper).toMatchSnapshot();
  });
  
});