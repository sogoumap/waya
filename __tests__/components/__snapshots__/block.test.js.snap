// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnapShot Cases Block 1`] = `
<div class="wayo-block">
  <i class="wayo-separator wayo-block__separator-top wayo-separator_direction_row wayo-separator_absolute" style="background-color: rgb(237, 237, 237); transform: scaleY(1);"></i>
  <i class="wayo-separator wayo-block__separator-bottom wayo-separator_direction_row wayo-separator_absolute" style="background-color: rgb(237, 237, 237); transform: scaleY(1);"></i>
</div>
`;

exports[`SnapShot Cases content 1`] = `<div class="wayo-block__content"></div>`;

exports[`SnapShot Cases header 1`] = `
<div class="wayo-block__header">
  <i class="wayo-separator wayo-block__separator-bottom wayo-separator_direction_row wayo-separator_absolute" style="background-color: rgb(237, 237, 237); transform: scaleY(1);"></i>
</div>
`;

exports[`SnapShot Cases title 1`] = `<div class="wayo-block__title"></div>`;
